code/modules/research/rdconsole.dm

```diff
@@ -423,10 +423,10 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 					message_admins("Protolathe exploit attempted by [key_name(usr, usr.client)]!")
 
 				if(g2g) //If input is incorrect, nothing happens
-					var/new_coeff = coeff * being_built.lathe_time_factor
-					var/time_to_construct = PROTOLATHE_CONSTRUCT_DELAY * new_coeff * amount ** 0.8
+					var/time_to_construct = PROTOLATHE_CONSTRUCT_DELAY * coeff * amount ** 0.8 //Construction time should correctly change with advanced components installed
 					var/enough_materials = 1
 
+					time_to_construct *= being_built.lathe_time_factor

```

Comment:
I know, mathematically, this is the same, but I prefer the way it reads prior to your implementation.

Do you have any specific reason why you did this?